content/metadata.yaml

```diff
@@ -23,6 +23,17 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2017-05-26
+  - github: delton137
+    name: Daniel C. Elton
+    orcid: 0000-0003-0249-1387
+    email: daniel.elton@nih.gov
+    affiliations:
+      - Radiology and Imaging Sciences, National Institutes of Health Clinical Center, Bethesda, MD
+    v1: ""
+    v2: "Revised specific sub-sections or supervised drafting one or more sub-sections"
+    coi:
+      string: "None"
+      lastapproved: !!str 2017-05-26
```

Comment:
Oh - one thing I just noticed - can you update this to the current date?

Reply:
Hi, I updated the "lastapproved" metadata...

but then I realized this metadata.yaml file in my PR is tagged as "outdated". Are you unable to merge this with the current version? 

let me know if you need me to do a pull to the latest version and then resubmit the PR. 

Thanks

Reply:
It looks like this will be able to be merged once the other one comes through. Thanks!